### PR TITLE
New Slovak translation

### PR DESCRIPTION
**Pull request by BIIINO**
Hello,

attached new translation for mobile APP - SLovak.

Thank you

### CODE REVIEW

**Comment by BIIINO** [comment]
And there is webapp version:

https://github.com/bytefury/crater/pull/428

**Review by komelabbbas — COMMENTED** [review]
Thanks, your slovak translation pr working perfectly.👌

Just need to change


`import sk from './sk.json';`
 instead of
`import ar from './sk.json';`

File: src/lang/i18n.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import es from './es.json';
 import de from './de.json';
 import it from './it.json';
 import ar from './ar.json';
+import ar from './sk.json';
```
[the remark this answers is not in the document]
Comment (BIIINO) [reply src/lang/i18n.js]: Hello, fixed :)

**Comment by BIIINO** [comment]
@komelabbbas  could you also before deploy check this issue please ? Its directly realted to slovakia, because in slovakia is separator comma instead of dot and if we want to use decimal prices in mobile app, we have to change region in IOS for example to US where in numeric keyboard is dot separator instead of comma.

issue here:

https://github.com/bytefury/crater-mobile/issues/15

Thank you a lot

**Comment by mohitpanjwani** [comment]
Hello @BIIINO Thank you for your contribution. 

We're looking into the given issue. 

Thanks

**Pull request merged**